pkg/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -50,9 +55,37 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// We only care about a configmap source with a specific name/namespace,
+	// so filter events before they are provided to the controller event handlers.
+	pred := predicate.Funcs{
+		UpdateFunc:  func(e event.UpdateEvent) bool { return handleConfigMap(e.MetaNew) },
```

Comment:
should that be `e.Meta` ?

Reply:
I tried e.Meta, then it failed as follows...
`clusterlogging_controller.go:61:73: e.Meta undefined (type event.UpdateEvent has no field or method Meta)`

Reply:
There is no `e.Meta` for `UpdateEvent` only `MetaOld` and `MetaNew` - see https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/event/event.go#L34